content/translations/contents.lr

```diff
@@ -0,0 +1,5 @@
+_hidden: yes
+---
+title: Translations in Progress
+---
+fontawesome: fa fa-language
```

Comment:
Note, this is using fontawesome _version 4_ (before they started enforcing weird licensing).